libraries/helper.rb

```diff
@@ -46,14 +46,24 @@ def omnibus_env
       node.run_state[:omnibus_env] ||= Hash.new { |hash, key| hash[key] = [] } # ~FC001
     end
 
+    # TODO: this method should just go away
     def omnibus_toolchain_enabled?
-      # We don't have an omnibus toolchain for windows yet.
-      !windows?
+      true
+    end
+
+    def toolchain_install_dir
+      if windows?
+        windows_safe_path_join(ENV['SYSTEMDRIVE'], 'opscode', node['omnibus']['toolchain_name'])
```

Comment:
Did we ever figure out why we *have* to hardcode `opscode` here? I'm ok with leaving it in the interest of getting this pushed forward, but maybe leaving a note saying it should be `chef` (or reasons etc)